src/listeners.js

```diff
@@ -214,6 +214,15 @@ export default class ListenerGenerator
     _attachComponentListeners() {
         this.componentListener = debounce((value) => {
             this._validate(value);
+
+            // Pass down errors to component if any
+            const errors = this.vm.errors.errors.filter(error => error.field === this.fieldName);
+
+            this.component.errors.clear();
```

Comment:
the error bag name isn't always `errors` the default is, however a user may specify different name when registering the plugin.